### ISSUE TITLE
Fonts in URL or tab bar are tiny

### ISSUE THREAD

**Issue by condor0765** [brave/browser-laptop#2469]
**Did you search for similar issues before submitting this one?**
Yes
**Describe the issue you encountered:**
Fonts in linux are tiny for the URL bar or tab bar.
**Expected behavior:**
Consistent with toolkit of the desktop environment
- Platform (Win7, 8, 10? macOS? Linux distro?):
  Xubuntu 14.04 LTS
- Brave Version:
  Brave: 0.11.0
  Electron: 1.2.3
  libchromiumcontent: 51.0.2704.103
  V8: 5.1.281.65
  Node.js: 6.1.0
  Update Channel: dev
- Screenshot if needed:
  The screenshot shows Brave and Firefox. While the URL bar and tab bar fonts in firefox are large (consistent) with my linux/openbox selection in Brave they are small.
  ![font](https://cloud.githubusercontent.com/assets/18568797/16807551/32f2b9cc-4919-11e6-8a6f-556a6845bf7b.png)

**Comment by bradleyrichter** [comment]
The fonts sizes were matched to chrome on Mac OS and are initially hard coded. We will need to carefully extend the layout to allow font size adjustment/adaptation to OS specific font sizes in the future without breaking the containers. (cc @bbondy )

**Comment by srirambv** [comment]
2 more from support 
https://linkbubble.zendesk.com/agent/tickets/7282
https://linkbubble.zendesk.com/agent/tickets/7291

**Comment by SimonCropp** [comment]
running high DPI on windows. nice browser but basically unusable when i cant read the fonts

**Comment by bsclifton** [comment]
@SimonCropp we just released a new version on Friday which you might try. I'm curious: what is your DPI setting at?

**Comment by sunworld23** [comment]
@bsclifton The tabs now look much better now

![capture](https://cloud.githubusercontent.com/assets/6208465/22627057/b2a7090e-eb88-11e6-88d4-299b164aa95f.PNG)

**Comment by SimonCropp** [comment]
Yep the latest build looks much better. Thanks

**Comment by bsclifton** [comment]
Awesome- DPI fix for Windows combined with an increase of the tab size (done with https://github.com/brave/browser-laptop/commit/615a6ec152a4535531a5cf46b7d6489d73060d62) should have resolved this issue 😄 

I'll close the issue for now, but please re-open if you believe the font sizes are hard to read
